### DISCUSSION TITLE
TE3 Measure do not display in PB  Desktop

### DISCUSSION THREAD

**Discussion by 10xape** [TabularEditor/TabularEditor3#1422] [Data Modeling]
Hi

I have created measure in the workspace mode and it does not display in visual in opposite to excact the same emasure created in Power BI Desktop connected directly to the model. What would be the possible issue?

-- Measure: [Prev_Cap_IntraDay]
-------------------------------
MEASURE '1_Measures'[Prev_Cap_IntraDay] =
    VAR CurrentLabel = SELECTEDVALUE( pc_cmc_cap_bridge_PQ2[time_label] )
    VAR CurrentToken = SELECTEDVALUE( pc_cmc_cap_bridge_PQ2[token_id] )
    VAR CurrentDate = SELECTEDVALUE( pc_cmc_cap_bridge_PQ2[refresh_date] )
    VAR PrevDayCap =
        CALCULATE(
            MAX( pc_cmc_cap_bridge_PQ2[cap_final] ),
            FILTER(
                ALL( pc_cmc_cap_bridge_PQ2 ),
                pc_cmc_cap_bridge_PQ2[token_id] = CurrentToken && pc_cmc_cap_bridge_PQ2[refresh_date] = CurrentDate - 1
                    && pc_cmc_cap_bridge_PQ2[time_label] = 6
            )
        )
    RETURN
        IF(
            CurrentLabel = 1,
            PrevDayCap,  -- Fallback to previous day's time_label = 6
            CALCULATE(
                MAX( pc_cmc_cap_bridge_PQ2[cap_final] ),
                FILTER(
                    ALL( pc_cmc_cap_bridge_PQ2 ),
                    pc_cmc_cap_bridge_PQ2[token_id] = CurrentToken && pc_cmc_cap_bridge_PQ2[refresh_date] = CurrentDate
                        && pc_cmc_cap_bridge_PQ2[time_label] = CurrentLabel - 1
                )
            )
        ),
    DisplayFolder = "1_measures\pc_cmc_cap_bridge_PQ2"

**Comment by otykier** [comment]
I'm sorry, but I don't understand what you're asking. Could you please clarify / add more details and perhaps a screenshot to help us understand the issue? Thank you.

**Reply by 10xape** [reply]
I edit report in direct connection to worspace sementic model in Power BI Desktop, I also have TE3 connected to the same worksapce model. Measure added in TE3 is not visable in the PI Desktop and I cannot add it to the visual.

**Reply by otykier** [reply]
After saving the changes from Tabular Editor (Ctrl+S), did you remember to press the Refresh button in Power BI Desktop?

![image](https://github.com/user-attachments/assets/321d8737-1b90-42a0-b796-dded534e21bc)

(The tooltip text here is misleading. Hitting this button doesn't actually perform a data refresh, since it's a live connection. Instead, the field list is updated to show any new measures, etc. that were added to the semantic model since the report was opened or the Refresh button was pushed last).
